hw03.py

```diff
@@ -0,0 +1,33 @@
+x = input("Write something:")
+if x.isdigit():
```

Comment:
You should use here `for ... in ...` to iterate through every character of the input string